### DISCUSSION TITLE
Glyph U+21E3 not rendering properly on MobaXterm

### DISCUSSION THREAD

**Discussion by jmigual** [ryanoasis/nerd-fonts#1374] [Q&A]
I've noticed recently that the symbol "⇣" does not render properly on MobaXterm. On Windows Terminal and VS Code's terminal it renders properly. 

Has anyone had this issue? If so, what are the solutions?

Some examples showing the sequence `[!⇣]`

MobaXterm:
![image](https://github.com/ryanoasis/nerd-fonts/assets/6493651/7d94c110-4a55-4cbc-b610-a5ab59f30355)

VS Code:
![image](https://github.com/ryanoasis/nerd-fonts/assets/6493651/5a624b9e-7f10-49fa-b564-2d2372ec5ab6)

Windows Terminal:
![image](https://github.com/ryanoasis/nerd-fonts/assets/6493651/eb8a097c-f287-4971-8d6f-62c2a81eb758)

**Comment by Finii** [comment]
https://onlinetools.com/unicode/convert-unicode-to-code-points tells us the codepoints:

![image](https://github.com/ryanoasis/nerd-fonts/assets/16012374/debb35dc-2788-4d8a-8382-02d3e1f1269f)

`21E3` is not patched in by Nerd Fonts. Either the original font has it, or the patched font will not have it either.

Most probably the terminals that show the glyph have some kind of fontfallback implemented, while the other one just shows glyphs present in the selected font.

**Reply by Finii** [reply]
List of glyphs / codepoints that Nerd Fonts patches is:

https://github.com/ryanoasis/nerd-fonts/wiki/Glyph-Sets-and-Code-Points
